src/NISysServer.py

```diff
@@ -20,6 +20,13 @@
 RESTART_MAX_RETRIES = 3
 RESTART_RETRY_DELAY = 1
 
+# Large string constants for XML responses
+SYSAPI_RESPONSE_LOGIN = "<?xml version='1.0' encoding='UTF-8'?><Permissions><Permission><Name>GetDB</Name><BuiltIn>false</BuiltIn><ID>0</ID></Permission><Permission><Name>SetDB</Name><BuiltIn>false</BuiltIn><ID>1</ID></Permission><Permission><Name>FSRead</Name><BuiltIn>false</BuiltIn><ID>2</ID></Permission><Permission><Name>FSWrite</Name><BuiltIn>false</BuiltIn><ID>3</ID></Permission><Permission><Name>SSLAdminModifyCerts</Name><BuiltIn>false</BuiltIn><ID>4</ID></Permission><Permission><Name>SSLAdminReadCerts</Name><BuiltIn>false</BuiltIn><ID>5</ID></Permission><Permission><Name>NIWebCer</Name><BuiltIn>false</BuiltIn><ID>6</ID></Permission><Permission><Name>GetWSAPIKey</Name><BuiltIn>false</BuiltIn><ID>7</ID></Permission><Permission><Name>ManageWS</Name><BuiltIn>false</BuiltIn><ID>8</ID></Permission><Permission><Name>SetWSAPIKey</Name><BuiltIn>false</BuiltIn><ID>9</ID></Permission><Permission><Name>WIFConfigureAppServer</Name><BuiltIn>false</BuiltIn><ID>10</ID></Permission><Permission><Name>GetSystemConfiguration</Name><BuiltIn>false</BuiltIn><ID>11</ID></Permission><Permission><Name>SetSystemConfiguration</Name><BuiltIn>false</BuiltIn><ID>12</ID></Permission><Permission><Name>FirmwareUpdate</Name><BuiltIn>false</BuiltIn><ID>13</ID></Permission><Permission><Name>Reboot</Name><BuiltIn>false</BuiltIn><ID>14</ID></Permission><Permission><Name>RemoteShell</Name><BuiltIn>false</BuiltIn><ID>15</ID></Permission><Permission><Name>SetRTLockPassword</Name><BuiltIn>false</BuiltIn><ID>16</ID></Permission><Permission><Name>ViewConsoleOutput</Name><BuiltIn>false</BuiltIn><ID>17</ID></Permission><Permission><Name>GetSyslog</Name><BuiltIn>false</BuiltIn><ID>18</ID></Permission><Permission><Name>ManageExtensions</Name><BuiltIn>false</BuiltIn><ID>19</ID></Permission></Permissions>"
+
+SYSAPI_RESPONSE_ENUM_EXPERTS = "<?xml version='1.0' encoding='utf-8' ?><NISysAPI_Results hr='0' version='00010001'><SystemExperts><ExpertInfo productID='{722D1D51-1C1D-42FE-AEEF-5CF6C711FFA7}' productDisplayName='NI System Configuration' productVersion='21.0.0d101' programmaticName='nisyscfg' /><ExpertInfo productID='{B831EF1F-BF94-42D7-9515-94A1D45F0E41}' productDisplayName='NI Systems Management' productVersion='21.1.0d492' programmaticName='nisysmgmt' /></SystemExperts></NISysAPI_Results>"
```

Comment:
For the experts, I don't think you should return anything but nisyscfg. Avoid nisysmgmt which refers to Skyline/SMS unless you're really trying to represent that it's there.

Also, use a simple version like "21.0" instead of the 'development' version you're including.

Reply:
Updated